safe-global/safe-wallet-web
# fix: add network switch warning to recovery execution

## What it solves

Resolves: https://github.com/safe-global/safe-wallet-web/pull/4162#issuecomment-2353248074

## How this PR fixes it
- Since the recovery tx is executed directly from the queue, we need a network warning to be shown at this point.
- Adds a network switch warning above the execute button.

## How to test it
- Create a recovery for another wallet.
- Open the safe with the recoverer wallet, wait for the recovery delay to elapse, then view the recovery transaction in the queue.
- If you're on the wrong network it should show a network switch warning, and the tooltip for the disabled execute button should show the relevant message.

## Screenshots
![image](https://github.com/user-attachments/assets/e0ab9c74-0e28-4f9e-9926-75c0d06fe46a)

## Checklist
